ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <react/renderer/components/view/ViewProps.h>
+
+namespace facebook {
+namespace react {
+
+class ActivityIndicatorProps final : public ViewProps {
+ public:
+  ActivityIndicatorProps() = default;
+  ActivityIndicatorProps(const ActivityIndicatorProps &sourceProps, const RawProps &rawProps);
+
+#pragma mark - Props
+  bool animating{true};
+  SharedColor color{};
```

Comment:
Are we are not setting default valur for color & size as specified in the react native document here ?